blog/docs/articles/knative-1.0.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Knative 1.0 is out!"
+linkTitle: "Knative 1.0 is out!"
+date: 2021-11-02
+description: "Details on the 1.0 release of Knative"
+type: "blog"
+---
+
+![1.0 Banner Image](/blog/images/1.0Banner.jpg)
+
+***Authors: [Carlos Santana](https://twitter.com/csantanapr) (IBM), [Omer Bensaadon](https://twitter.com/omer_bensaadon) (VMware), [Maria Cruz](https://twitter.com/marianarra_) (Google)***
+
+Today we release Knative 1.0, reaching an important milestone made possible thanks to the contributions and collaboration of over 600 developers. The Knative project was released by Google in July 2018, and it was developed in close partnership with VMWare, IBM, Red Hat, and SAP. Over the last 3 years, Knative has become [the most widely-installed serverless layer on Kubernetes.](https://www.cncf.io/wp-content/uploads/2020/11/CNCF_Survey_Report_2020.pdf)
+
+
+## What's new
+
+In case you haven't been following Knative's development closely, there have been a lot of changes since our initial release in July 2018.
+
+In addition to a myriad of bug fixes, stability and performance enhancements, our community has shipped the following improvements, in approximately chronological order:
+
+* Support for multiple HTTP routing layers beyond (including Istio, Contour, Kourier and Ambassador)
```

Comment:
```suggestion
* Support for multiple HTTP routing layers (including Istio, Contour, Kourier and Ambassador)
```

> Cheeky Istio first ;)

I actually forgot to remove the most important word.  The point was, it Istio-only at launch, and others were added later.